*** mmm-utils.el	Sat Nov 21 20:17:50 2009
--- mmm-utils.org	Sat Nov 21 21:46:54 2009
***************
*** 78,84 ****
      (save-match-data
        (dolist (pair arg-pairs)
          (while (string-match (car pair) string)
!           (setq string (replace-match (format-mode-line (cdr pair)) t t string))))))
    string)
  
  (defun mmm-format-matches (string &optional on-string)
--- 78,84 ----
      (save-match-data
        (dolist (pair arg-pairs)
          (while (string-match (car pair) string)
!           (setq string (replace-match (cdr pair) t t string))))))
    string)
  
  (defun mmm-format-matches (string &optional on-string)
